test/EventTest.php

```diff
@@ -6,12 +6,11 @@
 
 use Laminas\EventManager\Event;
 use Laminas\EventManager\Exception;
+use PHPUnit\Framework\Attributes\Group;
 use PHPUnit\Framework\TestCase;
 use stdClass;
 
-/**
- * @group      Laminas_Stdlib
- */
+#[Group('Laminas_Stdlib')]
```

Comment:
Can we grop the Group annotations completely - they're just noise

Reply:
Removed 👍 https://github.com/laminas/laminas-eventmanager/pull/61/commits/386056eff695fc72c6e7545d95d5a3fd9e9df148